res/css/views/messages/_CallEvent.scss

```diff
@@ -19,22 +19,24 @@ limitations under the License.
     width: 100%;
 
     .mx_CallEvent {
-        position: relative;
         display: flex;
         flex-direction: row;
+        flex-wrap: wrap;
         align-items: center;
         justify-content: space-between;
+        gap: 4px 0;
 
+        position: relative;
+        margin: 4px 0;
+        padding: 12px 24px;
```

Comment:
we should probably use our spacing variables now that we have them

Reply:
I see. I am wondering what the policy on using and applying those variables is. Will they be applied gradually and incrementally to see how they should work, or at one time globally?